x-pack/plugins/triggers_actions_ui/kibana.json

```diff
@@ -3,5 +3,5 @@
     "version": "kibana",
     "server": false,
     "ui": true,
-    "requiredPlugins": ["management", "charts", "data"]
+    "requiredPlugins": ["management", "charts", "data", "alerting", "alertingBuiltins"]
```

Comment:
I'm wondering if these should be optional dependencies? This would allow the scenario the alerting plugin is disabled but actions / connectors plugin is enabled. (UI would still show)

Reply:
Ya, that makes sense.  I think for both of these, I ended up moving some code to common, that this plugin imported, so wanted to note the dependency, somehow.  Of course, it doesn't matter if those deps aren't enabled :-). Optional makes more sense.